packages/rpm/centos/mysql56-community-mroonga.spec.in

```diff
@@ -91,6 +91,12 @@ mv $RPM_BUILD_ROOT%{_datadir}/doc/mroonga/ mysql-mroonga-doc/
 rm -rf $RPM_BUILD_ROOT
 
 %post
+/sbin/service mysqld status
+if [ $? -ne 0 ] ; then
+  /sbin/service mysqld start
+  stop_after_installation=1
+fi
```

Comment:
Ensure initializing `stop_after_installation` to avoid effects from environment variable. Here is a sample implementation:

```sh
else
  stop_after_installation=0
```